docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
     tty: true
     command: ./entrypoint.sh
   api:
+    platform: linux/x86_64
```

Comment:
Required to run on arm64 machines (Mac chips). Otherwise compose results in some error like `no matching manifest for linux/arm64/v8 in the manifest list entries`.

Or maybe its just required to run on my computer 🤷. Happy to remove this if it causes problems